src/actions.js

```diff
@@ -86,7 +87,7 @@ export function fetchOrganisations() {
     fetch(url, opts)
       .then(responseObj => responseObj.json())
       .then(responseData => {
-        const organisation = state.organisations.selected;
+        const selectedOrganisationLocalStorage = getLocalStorage("lizard-management-current-organisation", null);
```

Comment:
I don't get why you need to get the selected organisation from the local storage and how is this change related to this issue to show the storage of rasters and scenarios. Does it fix have anything to do with this bug: https://github.com/nens/lizard-management-client/issues/680?

Reply:
Also, this leads to bug. If I don't have any selected organisation in the local storage, it will cause error: `Cannot read property 'roles' of null`. So overall, I think this is not a good change.

Reply:
You are right!
I need to look into this.
I will start to merge the fix for the 680 bug to see if it helps.

Reply:
@hoanphungt 
Hi Hoan,
I reproduced your error and believe I fixed it.
Also in the meanwhile I found bugs with the routing that I also fixed (I think so).
The "getLocalStorage("lizard-management-current-organisation", null); " here is needed to retrieve the selected organisation that the user selected during last usage of the app.
Previously this was done in the reducer default value, but here it is easier to also trigger the 'usage' api on startup. This usage api is fetched anytime a selected organisation is changed through the action. (but not if it is the default of the redux store).